### PR TITLE
New package: xdebug-2.3.1

### PR DESCRIPTION
**Pull request by etosan**
My first attempt at packaging something publicly. 

Sorry if I fucked up, I would like to request somebody more advanced to review this ... as I am idiot. Distfile hash was crosschecked against my Arch Linux install but nothing more.
That's why revision is set  to 0.

As usual I have no clue.

Upstream project http://xdebug.org/

### CODE REVIEW

File: srcpkgs/xdebug/template [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Template build file for 'xdebug'
+pkgname=xdebug
+version=2.3.1
+revision=0
+short_desc="Debugger and Profiler Tool for PHP"
+license="Xdebug"
```
Comment (Gottox) [inline srcpkgs/xdebug/template]: Is the LICENSE file installed?

File: srcpkgs/xdebug/template [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	phpize
+}
+
+do_configure() {
+	./configure --enable-xdebug
+}
+
+do_build() {
+	make
+}
```
Comment (Gottox) [inline srcpkgs/xdebug/template]: Have you considered using gnu-configure instead of defining do_{configure,build,install}?

Comment (xtraeme) [inline srcpkgs/xdebug/template]: what @Gottox said

**Comment by Gottox** [comment]
What's the state of this PR?

**Comment by bougyman** [comment]
Closing for non-response.

**Pull request closed without merging**